Fair (#41)
S-ENDA/DMH: documentation/fair.md

## FAIR

> The [FAIR](https://www.go-fair.org/fair-principles/) guiding principles for scientific data management illustrate the best practice for sharing open data. They were first set out in the 2016 paper [The FAIR Guiding Principles for scientific data management and stewardship](https://doi.org/10.1038/sdata.2016.18). The principles emphasise machineactionability (i.e., the capacity of computational systems to find, access, interoperate, and reuse data with none or minimal human intervention) because humans increasingly rely on computational support to deal with data as a result of the increase in volume, complexity, and creation speed of data.

Reviewer: ```suggestion
The [FAIR](https://www.go-fair.org/fair-principles/) guiding principles for scientific data management illustrate the best practice for sharing open data. They were first set out in the 2016 paper [The FAIR Guiding Principles for scientific data management and stewardship](https://doi.org/10.1038/sdata.2016.18). The principles emphasise machine-actionability (i.e., the capacity of computational systems to find, access, interoperate, and reuse data with none or minimal human intervention) because humans increasingly rely on computational support to deal with data as a result of the increase in volume, complexity, and creation speed of data.
```

|Initials|Description|
|:----|:----|
| **F**indable | The first step in (re)using data is to find them. Metadata and data should be easy to find for both humans and computers. This includes unique and persistent identifiers, rich metadata and that the (meta)data is registered in a searchable resource.|
|**A**ccessible | Once the user finds the required data, they need to know how they can be accessed. This includes using a standarised communications protocol and where necessary authentication and authorisation.|
|**I**nteroperable | The data usually need to be integrated with other data. In addition, the data need to interoperate with applications or workflows for analysis, storage, and processing. This includes FAIR vocabularies.|

> |**R**eusable | The ultimate goal of FAIR is to optimise the reuse of data. To achieve this, metadata and data should be well-described so that they can be replicated and/or combined in different settings. This includes accurate and relevant attributes, data usage license, detailed provenance and meeting domain-relevant community standards.|

Reviewer: ```suggestion
|**R**eusable | The ultimate goal of FAIR is to optimise the reuse of data. To achieve this, metadata and data should be well-described so that they can be replicated and/or combined in different settings. This includes accurate and relevant attributes, data usage license, detailed provenance, and meeting domain-relevant community standards.|
```

### FAIR in S-ENDA and GeoNorge
All metadata and datasets that the S-ENDA project delivers to Geonorge is assessed through FAIR-indicators. The status of various datasets' fulfillment of the FAIR principles as well as other requirements from the national geographic infrastructure can bee seen in the [Mareano status register](https://register.geonorge.no/mareano-statusregister). This status register is a first version and will be improved over time. Assessment is done for each of the four main groups in FAIR and it is possible to achieve 100% for each of the letters (F-A-I-R). The method is based on the same implementation that is used in national and

> European portals for open data, but criteria are closely linked to requirements from the national infrastructure for geographic information especially in terms of accessibility (A) and interoperability (I).

Reviewer: ```suggestion
European portals for open data, but criteria are closely linked to requirements from the national infrastructure for geographic information, especially in terms of accessibility (A) and interoperability (I).
```
